From 1b1c99626404328b571e1f6c18b50f9bed9c2e2c Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Tue, 28 Aug 2012 09:08:45 +0200
Subject: [PATCH 321/366] usb-redir: Get rid of unused async-struct dev member

This is a preparation patch for completely getting rid of the async-packet
struct in usb-redir, instead relying on the (new) per ep queues in the
qemu usb core.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 hw/usb/redirect.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/hw/usb/redirect.c b/hw/usb/redirect.c
index 816a19a..4a23b82 100644
--- a/hw/usb/redirect.c
+++ b/hw/usb/redirect.c
@@ -96,7 +96,6 @@ struct USBRedirDevice {
 };
 
 struct AsyncURB {
-    USBRedirDevice *dev;
     USBPacket *packet;
     uint32_t packet_id;
     QTAILQ_ENTRY(AsyncURB)next;
@@ -255,7 +254,6 @@ static int usbredir_write(void *priv, uint8_t *data, int count)
 static AsyncURB *async_alloc(USBRedirDevice *dev, USBPacket *p)
 {
     AsyncURB *aurb = (AsyncURB *) g_malloc0(sizeof(AsyncURB));
-    aurb->dev = dev;
     aurb->packet = p;
     aurb->packet_id = dev->packet_id;
     QTAILQ_INSERT_TAIL(&dev->asyncq, aurb, next);
-- 
1.7.12

